modules/attribution-document/attribution-document-core/src/main/java/org/eclipse/sw360/antenna/attribution/document/utils/TemplateLoaderUtil.java

```diff
@@ -32,9 +33,57 @@ private TemplateLoaderUtil() {
     }
 
     public static Templates load(String key) {
-        TemplateBundle bundle = getLoader(key);
+        return load(getLoader(key));
+    }
+
+    public static Templates load(File cover, File copyright, File content, File back) {
+        return load(new TemplateBundle() {
+            @Override
+            public String key() {
+                return "";
+            }
+
+            @Override
+            public InputStream loadTitleTemplate() {
+                try {
+                    return new FileInputStream(cover);
```

Comment:
Where are these streams closed? I couldn't find code / a caller that closes these.

Reply:
These are closed here: https://github.com/eclipse/antenna/blob/master/modules/attribution-document/attribution-document-core/src/main/java/org/eclipse/sw360/antenna/attribution/document/core/Templates.java#L115-L141